src/common/ConfigReader.cpp

```diff
@@ -27,19 +27,31 @@
 #include <QtCore/QMap>
 #include <QtCore/QBuffer>
 #include <QtCore/QFileInfo>
+#include <QtCore/QtGlobal>
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+#include <QtCore/QStringTokenizer>
+#endif
+#include <QtCore/QStringView>
 
 QTextStream &operator>>(QTextStream &str, QStringList &list)  {
     list.clear();
 
     QString line = str.readLine();
-
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+    const auto strings = QStringTokenizer{line, u','};
```

Comment:
Is it worth it to use `QStringTokenizer` here or would `QStringView::split` work well for both?